src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -52,6 +53,12 @@ CONTAINER_OOM_ERROR=196
 CONTAINER_OOD_ERROR=198
 RUNTIME_UNRECOGNIZED_ERROR=255
 
+SIG_2="SIGINT"
+SIG_6="SIGABRT"
+SIG_9="SIGKILL"
+SIG_11="SIGSEGV"
+SIG_15="SIGTERM"
```

Comment:
Seems you only need to exit with the exitcode, and no need to care about the string represent if the exitcode?
So, remove them for easy to maintain?

Reply:
I have removed hard code signal number and just maintain the signal name.